docs/snapshotting/snapshot-support.md

```diff
@@ -9,6 +9,12 @@ guest workload at that particular point in time.
 
 ## Snapshotting in Firecracker
 
+### Supported platforms
+
+The Firecracker snapshot feature is in [developer preview](docs/RELEASE_POLICY.md) on all CPU micro-architectures listed in
+[README](../README.md#supported-platforms) except ARM which is not supported.
```

Comment:
I don't think we need to specify that, since Arm is already marked as dev preview in the referenced section. And, if we keep it here it's one more thing to remember deleting later.

Reply:
ARM is supported as dev preview, but dev preview snapshots do not work on ARM in  v0.23